tools/provision/chocolatey/zstd.ps1

```diff
@@ -0,0 +1,76 @@
+#  Copyright (c) 2014-present, Facebook, Inc.
+#  All rights reserved.
+#
+#  This source code is licensed under the BSD-style license found in the
+#  LICENSE file in the root directory of this source tree. An additional grant
+#  of patent rights can be found in the PATENTS file in the same directory.
+
+# Update-able metadata
+$version = '1.2.0'
+$chocoVersion = '1.2.0-r1'
+$packageName = 'zstd'
+$projectSource = 'https://github.com/facebook/zstd'
+$packageSourceUrl = 'https://github.com/facebook/zstd'
+$authors = 'Facebook'
+$owners = 'Facebook'
+$copyright = 'https://github.com/facebook/zstd/blob/master/LICENSE'
+$license = 'https://github.com/facebook/zstd/blob/master/LICENSE'
+$url = "https://github.com/facebook/zstd/archive/v$version.zip"
+
+# Invoke our utilities file
+. "$(Split-Path -Parent $MyInvocation.MyCommand.Definition)\osquery_utils.ps1"
+
+# Invoke the MSVC developer tools/env
+Invoke-BatchFile "$env:VS140COMNTOOLS\..\..\vc\vcvarsall.bat" amd64
+
+# Time our execution
+$sw = [System.Diagnostics.StopWatch]::startnew()
+
+# Keep the location of build script, to bring with in the chocolatey package
+$buildScript = $MyInvocation.MyCommand.Definition
+
+# Create the choco build dir if needed
+$buildPath = Get-OsqueryBuildPath
+if ($buildPath -eq '') {
+  Write-Host '[-] Failed to find source root' -foregroundcolor red
+  exit
+}
+$chocoBuildPath = "$buildPath\chocolatey\$packageName"
+if (-not (Test-Path "$chocoBuildPath")) {
+  New-Item -Force -ItemType Directory -Path "$chocoBuildPath"
+}
+Set-Location $chocoBuildPath
+
+# Retreive the source
+Invoke-WebRequest $url -OutFile "$packageName-$version.zip"
```

Comment:
Would you mind changing this to read:
```
$zipFile = "$packageName-$version.zip"
if(-not (Test-Path $zipFile)) {
  Invoke-WebRequest $url -OutFile $zipFile
}
```
So we prevent re-downloading the source if we've already got it?